sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/SyncActivityViewModel.kt

```diff
@@ -86,4 +94,54 @@ class SyncActivityViewModel @Inject constructor(
             ),
         )
     }
+
+    fun onTurnOffSyncConfirmed() {
+        viewModelScope.launch(dispatchers.io()) {
+            viewState.emit(viewState.value.copy(showAccount = false))
+            val deviceId = syncRepository.getThisConnectedDevice().deviceId
+            when (syncRepository.logout(deviceId)) {
+                is Error -> {
+                    updateViewState()
```

Comment:
this is the same for both cases, do we need that distinction?

Reply:
Makes sense, but I've checked branch on top of this one and it will change, so I'm going to leave it as it is now.